From dd752664a04dc71aa4e3c59620ec69b822af6be1 Mon Sep 17 00:00:00 2001
From: Zhe Zhang <zhz@apache.org>
Date: Tue, 28 Feb 2017 09:00:49 -0800
Subject: [PATCH 2319/2848] HDFS-11466. Change
 dfs.namenode.write-lock-reporting-threshold-ms
 default from 1000ms to 5000ms. Contributed by
 Andrew Wang.

(cherry picked from commit d269b488a71a158d3ddcbdea96992abe29725c69)
(cherry picked from commit 5fb785e154b429beb0b2e392c39be91dae65bb5c)
(cherry picked from commit 265ddb20c3aa99c2d91e2a85e82106d4960f5a7b)
(cherry picked from commit 33f4263345087d0b133ee3aba86b0ddfe239d81b)

Change-Id: I96077847b60ccab163cf2ac41f59b44cac616f50
---
 .../java/org/apache/hadoop/hdfs/DFSConfigKeys.java |    2 +-
 .../src/main/resources/hdfs-default.xml            |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java
index a0d7ca3..e1fa702 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSConfigKeys.java
@@ -416,7 +416,7 @@
   // event to be logged
   public static final String  DFS_NAMENODE_WRITE_LOCK_REPORTING_THRESHOLD_MS_KEY =
       "dfs.namenode.write-lock-reporting-threshold-ms";
-  public static final long    DFS_NAMENODE_WRITE_LOCK_REPORTING_THRESHOLD_MS_DEFAULT = 1000L;
+  public static final long    DFS_NAMENODE_WRITE_LOCK_REPORTING_THRESHOLD_MS_DEFAULT = 5000L;
   public static final String  DFS_NAMENODE_READ_LOCK_REPORTING_THRESHOLD_MS_KEY =
       "dfs.namenode.read-lock-reporting-threshold-ms";
   public static final long    DFS_NAMENODE_READ_LOCK_REPORTING_THRESHOLD_MS_DEFAULT = 5000L;
diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml b/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
index 1bc3cad..e3c08e2 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml
@@ -2292,7 +2292,7 @@
 
 <property>
   <name>dfs.namenode.write-lock-reporting-threshold-ms</name>
-  <value>1000</value>
+  <value>5000</value>
   <description>When a write lock is held on the namenode for a long time,
     this will be logged as the lock is released. This sets how long the
     lock must be held for logging to occur.
-- 
1.7.9.5

